bentoml/cli/__init__.py

```diff
@@ -144,6 +145,23 @@ def deploy(archive_path, platform, region, stage):
                 'Deploying with "--platform={platform}" is not supported in the current version of BentoML'
                 .format(platform=platform))
 
+    # Example useage: bentoml delete-deployment SERVICE_NAME --platform=aws-lambda
+    @_cli.command()
+    @click.argument('service-name', type=click.STRING)
+    @click.option('--platform', type=click.Choice([
+        'aws-lambda', 'aws-lambda-py2', 'gcp-function', 'aws-sagemaker', 'azure-ml', 'algorithmia'
+    ]), required=True)
+    @click.option('--region', type=click.STRING, required=True)
```

Comment:
will the `bentoml deploy` command fail if it has already been deployed to a different region in the same account? for simple deployment I thought it might be easier to just use the service name as "key" for delete and status

Reply:
check out the comment below.  I think your next comment raise a good point. Let's use archive_path for now.